app/views/users/new.html.erb

```diff
@@ -1,7 +1,20 @@
-<% provide(:title, '.title') %>
+<%= provide :title, t(".title") %>
 <div class="center jumbotron">
-  <h2>
-    <%= t ".sign_up_section" %>
-    <%= link_to t(".sample_app"), Settings.link_rail %>
-  </h2>
+  <h1><%= t ".title" %></h1>
+  <div class="row">
+    <div class="col-md-6 col-md-offset-3">
+      <%= form_for @user do |f| %>
```

Comment:
phân biệt form for và các loại form khác

Reply:
- form_for : Tạo form khi có một đối tượng, model cụ thể, thường dùng khi tạo mới hay update ,tự động thêm id, class cho cả thẻ form và các form field
- form_tag : Tạo form không có đối tượng , tự động thêm id, class cho các form field
- form_with : Tạo form có đối tượng hay không đều được, tránh sai cú pháp , các form field không cần phải tương ứng với các thuộc tính của model